docs/pages/api-docs/x-grid.md

```diff
@@ -46,9 +46,18 @@ import { XGrid } from '@material-ui/x-grid';
 | <span class="prop-name">logLevel</span> | <span class="prop-type">string | false</span> | false | Allows to pass the logging level or false to turn off logging. |
 | <span class="prop-name">nonce</span> | <span class="prop-type">string</span> |   | Nonce of the inline styles for [Content Security Policy](https://www.w3.org/TR/2016/REC-CSP2-20161215/#script-src-the-nonce-attribute). |
 | <span class="prop-name">onCellClick</span> | <span class="prop-type">(param: GridCellParams) => void</span> |   | Callback fired when a click event comes from a cell element. |
-| <span class="prop-name">onCellHover</span> | <span class="prop-type">(param: GridCellParams) => void</span> |   | Callback fired when a hover event comes from a cell element. |
+| <span class="prop-name">onCellDoubleClick</span> | <span class="prop-type">(param: GridCellParams) => void</span> |   | Callback fired when a double click event comes from a cell element. |
```

Comment:
No mention of the second event argument?

Reply:
ah yes